hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/commit/ITestAbfsRenameStageFailure.java

```diff
@@ -41,6 +42,12 @@ public ITestAbfsRenameStageFailure() throws Exception {
     binding = new ABFSContractTestBinding();
   }
 
+  protected boolean isNamespaceEnabled() {
+    FileSystem fs = getFileSystem();
+    String namespaceEnabled = fs.getConf().get("fs.azure.test.namespace.enabled");
+    return namespaceEnabled.equals("true");
```

Comment:
[nit]:("true").equals(namespaceEnabled) is better since, namespaceEnabled can be null.

Reply:
Addressed in latest commit.